pilot/pkg/networking/core/v1alpha3/route/route.go

```diff
@@ -1248,8 +1233,8 @@ func consistentHashToHashPolicy(consistentHash *networking.LoadBalancerSettings_
 	return nil
 }
 
-func getHashForService(node *model.Proxy, push *model.PushContext,
-	svc *model.Service, port *model.Port,
+func hashForService(node *model.Proxy, push *model.PushContext, svc *model.Service,
```

Comment:
nit: keep consistent of the order with other functions 1. push context, 2. proxy, 3 others (svc, port)